src/lib/reducers/theme.js

```diff
@@ -2,22 +2,40 @@ const SET_THEME = 'SET_THEME'
 const DEFAULT_THEME = 'default'
 const PRIMARY_THEME = 'primary'
 const THEMES = [DEFAULT_THEME, PRIMARY_THEME]
+const EMPTY_OVERRIDES = {}
 
-// action creator
-export const setTheme = theme => ({
+/**
+ * Change the cozy-bar theme
+ * 
+ * Today, the value 'primary' will change the background color
+ * of the bar in the mobile view. It will then use the 
+ * `--primaryColor` CSS variable and the `--primaryContrastTextColor`
+ * for the text.
+ * 
+ * @function
+ * @param {String} theme - either 'default' or 'primary'
+ * @param {Object} overrides - overrides of default values for the theme
+ *                             default to an empty object (no overrides)
+ *                             It will only overrides the values for the 
+ *                             'primary' specific theme/view
+ * @param {Object} overrides.primaryColor - the background color
+ * @param {Object} overrides.primaryContrastTextColor - the text color
+ * @returns {Object} new state like `{ type: 'default', overrides: {} }`
+ */
+export const setTheme = (theme=null, overrides=EMPTY_OVERRIDES) => ({
   type: SET_THEME,
-  theme
+  theme: { type: theme, overrides }
 })
 
 // reducer
-export const getDefaultTheme = () => DEFAULT_THEME
+export const getDefaultTheme = () => ({type: DEFAULT_THEME, overrides: EMPTY_OVERRIDES})
```

Comment:
I wonder if it would not be simpler to have

```
DEFAULT_THEME = { type: 'default', overrides: {} }
```

not to rebuild the theme object by hand when we need it (also on L38)

Reply:
Also `name` could be more appropriate than `type` here since we already have `type` in the action and it can be confusing.